packages/components/src/InputValidation/InputValidation.mdx

```diff
@@ -24,27 +24,23 @@ import { InputValidation } from "@jobber/components/InputValidation";
 
 <Playground>
   {() => {
-    const [value, setValue] = useState("Juan");
     const [validationMessages, setValidationMessages] = useState(undefined);
     return (
       <>
-        <InputValidation messages={validationMessages} />
+        <InputValidation message={validationMessages} />
         <Text>
           My name is
           <InputText
-            validations={[
-              {
-                shouldShow: value !== "Jeff",
-                message: "Have you considered a better name, like Jeff?",
-                status: "error"
+            validations={{
+              pattern: {
+                value: /Jeff/g,
```

Comment:
Should we have the `g` in here? I think we want to match just a single `Jeff` no?